### PR TITLE
Reload if server says so

### PR DESCRIPTION
**Pull request by diocas**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 $(document).ready(function() {
 	if (OC.currentUser !== false) {
 		$.getJSON("/index.php/apps/canary", function(data) {
-			if (data["is_adopter"]) {
+			if (data["reload"]) {
```
Comment (labkode) [inline js/app.js]: Keep still the is_adopter check for the transition, should not harm

Comment (labkode) [reply js/app.js]: if (is_adopter OR reload) ...

Comment (diocas) [reply js/app.js]: done

**Comment by diocas** [comment]
can merge

**Review by labkode — APPROVED** [review]

**Pull request merged**